app/experimenter/nimbus-ui/schema.graphql

```diff
@@ -47,6 +47,8 @@ input ExperimentInput {
   totalEnrolledClients: Int
 }
 
+scalar JSONString
```

Comment:
The frontend won't know what to do with this, but we can map it to a type [here](https://github.com/mozilla/experimenter/blob/main/app/experimenter/nimbus-ui/src/react-app-env.d.ts#L7:L7). Something like `type JSONString = Record<any, any>;`

Reply:
Oh yeah... it's doing that because we store the entire DTO on the changelog but we won't need it in the frontend until we build out the changelog UI which is coming soon in an epic so we can probably deal with it then.  But good call out 👍